persistence/sql/operational-scripting.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Operational Scripting
+summary: Explains how to generate scripts for SQL Persistence
+component: SqlPersistence
+reviewed: 2021-02-19
+---
+
+## Operational Scripting
+
+In order to generate database scripts outside of a build process, the command prompt tool developed for this purpose can be used.
+
+The tool can be obtained from NuGet and installed using the following command:
+
+```
+dotnet tool install -g NServiceBus.Persistence.Sql.CommandLine
+```
+
+Once installed, the `sql-persistence` command line tool will be available for use.
+
+`sql-persistence <command> [options]`
+
+### Available commands
+
+- `script`
```

Comment:
This section seems confusing with only one command. Should we explicitly say there is only one command?

Reply:
I thought about it, but thought to keep the format the same as other command line documentations, and this will  be expanded (hopefully) in the future.

Maybe explicitly saying that for now it has only one command?